src/test/java/seedu/address/storage/JsonAdaptedCompanyTest.java

```diff
@@ -28,10 +29,12 @@
     private static final String VALID_PHONE = BENSON.getPhone().toString();
```

Comment:
Not an issue but should we refractor the person variable to company variable. Instead of benson, use like meta or something?

Reply:
Done as suggested 👍